superset-frontend/src/SqlLab/components/SouthPane.jsx

```diff
@@ -59,6 +60,29 @@ const defaultProps = {
   offline: false,
 };
 
+const StyledPane = styled.div`
+  width: 100%;
+
+  .ant-tabs .ant-tabs-content-holder {
+    overflow: visible;
+  }
+
+  .SouthPaneTabs {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+  }
+  .tab-content {
+    .alert {
+      margin-top: 10px;
```

Comment:
```suggestion
      margin-top: ${({ theme }) => theme.gridUnit * 2}px;
```
It's ideal to move hard-coded pixel gutters/margins/paddings/spacings like this to using the `gridUnit` from the Superset theme. In cases like this where it's been 10px for a long time, we should use our best judgment as to whether 2 units (8px) or 3 units (12px) looks more consistent. 8px is more common, so I took a guess here :)